tests/CMakeLists.txt

```diff
@@ -35,7 +35,6 @@ endfunction(add_header_test)
 add_header_test(cl_h test_cl.h.c)
 add_header_test(cl_egl_h test_cl_egl.h.c)
 add_header_test(cl_ext_h test_cl_ext.h.c)
-add_header_test(cl_ext_intel_h test_cl_ext_intel.h.c)
```

Comment:
We should keep testing the file as long as it exists.

Reply:
I've added back testing for this file.  Note, this means that the message about extensions being moved is emitted when tests are built, as expected.  I just want to confirm this behavior is OK:

```sh
[ 20%] Building C object CMakeFiles/cl_ext_intel_h_300.dir/test_cl_ext_intel.h.c.o
In file included from /home/bashbaug/git/OpenCL-Headers/tests/test_cl_ext_intel.h.c:19:0:
/home/bashbaug/git/OpenCL-Headers/tests/../CL/cl_ext_intel.h:19:9: note: #pragma message: The Intel extensions have been moved into cl_ext.h.  Please include cl_ext.h directly.
 #pragma message("The Intel extensions have been moved into cl_ext.h.  Please include cl_ext.h directly.")
         ^~~~~~~
[ 21%] Linking C executable cl_ext_intel_h_300
```

Reply:
Yes, that's fine.